### ISSUE TITLE
marker.setDisableAutoPan(true) does not work on iOS

### ISSUE THREAD

**Issue by aMUSiC** [mapsplugin/cordova-plugin-googlemaps#1918]
**I'm submitting a ...**  (check one with "x")
[ ] question
[x] any problem or bug report
[ ] feature request

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
[ ] **Android**
[x] **iOS**

**Current behavior:**

setting marker.setDisableAutoPan(true) does not stop from the map panning whenever a marker is clicked.

**Expected behavior:**

marker.setDistableAutoPan(true) should prevent the map from panning upon marker click.

**Additional Info:**

marker.setDisableAutoPan(true) works as expected on Android 7.0 but not on iOS 11 (on the same project);

No errors on the xcode console.

Using Ionic 3.9.2, cordova-plugin-googlemaps 2.1.1 and @ionic-native/google-maps 4.4.2

**Comment by wf9a5m75** [comment]
use the multiple_maps branch version. It's already fixed.

**Comment by wf9a5m75** [comment]
ping

**Comment by aMUSiC** [comment]
I will try the multiple_maps branch then. Thank you.

**Comment by aMUSiC** [comment]
Unfortunately no matter what I tried I can't seem to be able to install the multiple_maps branch. I checked all the issue tickets about that on the branch (updated npm etc), except downgrading cordova (which is not an option on the particular project, especially since it's getting automatically built through a CI server). I guess I'll have to wait until the multiple_maps branch makes it into master then...

**Comment by wf9a5m75** [comment]
Try like this:

```
$> git clone https://github.com/mapsplugin/cordova-plugin-googlemaps
$> git checkout multiple_maps
$> git pull

$> cd (path to your project folder)
$> cordova plugin rm cordova-plugin-googlemaps
$> cordova plugin (path to cordova-plugin-googlemaps folder) --variable API_KEY_FOR_ANDROID=".." --variable API_KEY_FOR_IOS=".."
```

**Comment by aMUSiC** [comment]
Installation works this way, and I can find a way to include it in CI. One thing I noticed though is that after installing it, the geolocation native plugin  (https://github.com/apache/cordova-plugin-geolocation) has stopped working (never manages to get a position, or throw an error even when a timeout is set in its options). Not sure if there's a conflict between the two plugins (at least on Android).

Will report later how this fares on ios (with respect to this issue's title).

**Comment by aMUSiC** [comment]
I'm really sorry for expanding the scope of this issue so much, but apparently now with multiple_maps I can't scroll the map or click on any markers. I know it's probably something to do with how my html/dom/css is set up, but the same setup was working fine with the master branch.

Is there any document online outlining changes and scope of the multiple_maps branch?

**Comment by aMUSiC** [comment]
Alright, It seems that if you render a div that covers the map and have it removed through *ngIf or elementRef when the map loads, the map won't receive the touch events in multiple_maps.

But now that I fixed all the little things that prevented my map from working, I managed to run multiple_maps on ios and..

.setDisableAutoPan(true) still does not work on iOS.. =(

**Comment by wf9a5m75** [comment]
```
map.addMarker({
  disableAutoPan:true
})
```

**Comment by wf9a5m75** [comment]
If you have problems, please share your project files on GitHub.

**Comment by aMUSiC** [comment]
confirmed. Adding it as an option in addMarker it works on iOS.

**Comment by aMUSiC** [comment]
And after snooping through Common.js, I figured out that I can add the "pgm-ignore" class on any dom element that I draw over the map (such as a cover to hide the map until it's done loading), and the map will then accept all touches normally. Nice! That's definitely a piece of information that needs to be added in the documentation, once multiple_maps is merged into master.
